client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -125,6 +126,7 @@ export default function getThankYouPageUrl( {
 	jetpackTemporarySiteId,
 	adminPageRedirect,
 	domains,
+	isGiftPurchase = false,
```

Comment:
Minor, but setting the default here is not necessary. If it is undefined, the condition below will still evaluate correctly.

```suggestion
	isGiftPurchase,
```